app/services/application_data_service.rb

```diff
@@ -0,0 +1,45 @@
+class ApplicationDataService
```

Comment:
I like this thing. Did you consider wrapping an ApplicationChoice in it, and extending that object's interface? See conversation with @tijmenb here: https://github.com/DFE-Digital/apply-for-teacher-training/pull/3156. Then instantiate one of these at the top of SingleApplicationPresenter, ApplicationDataExport etc?